### PR TITLE
Artist details page removed

### PR DESCRIPTION
**Pull request by hedgehog1833**
Artist details page and everything around it removed

### CODE REVIEW

**Comment by hedgehog1833** [comment]
[Remove artist detail page and all corresponding stuff](https://trello.com/c/H63udfs4/106-remove-artist-detail-page-and-all-corresponding-stuff)

**Comment by hedgehog1833** [comment]
[404 on /my-artists](https://trello.com/c/svlco9wg/114-404-on-my-artists)

File: src/main/resources/static/css/frontend.css [hunk 1]
```diff
@@ -275,3 +275,7 @@ footer{
     border-radius: .25rem;
     border: 1px solid #ced4da;
 }
+
+.follow-artist-button {
+    margin-top: 0.5rem;
+}
```
Comment (DanielW1987) [inline src/main/resources/static/css/frontend.css]: you can use bootstrap class for this (mt-1 to mt-5).

File: src/main/resources/static/js/util.js [hunk 2]
```diff
@@ -47,7 +47,7 @@ function createFollowArtistButton(artistName,artistId,isFollowed) {
   const followArtistButtonElement = document.createElement('button');
   followArtistButtonElement.id = "followArtistButton"+artistId;
   followArtistButtonElement.type = "button";
-  followArtistButtonElement.className = "btn btn-primary btn-dark font-weight-bold";
+  followArtistButtonElement.className = "btn btn-primary btn-dark font-weight-bold follow-artist-button";
```
Comment (DanielW1987) [inline src/main/resources/static/js/util.js]: don't forget to remove the class here...

**Pull request merged**